pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -110,6 +110,15 @@ final class Event {
           if (removed != null) 'removed': removed,
         };
 
+  /// An event that reports when the code size measurement is run
+  /// via `--analyze-size`.
+  ///
+  /// [kind] - string identifier for which platform was run "ios", "apk",
```

Comment:
Is this the `platform` field from https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/reporting/events.dart#L237 ?

Reply:
Ah yep, good catch